### ISSUE TITLE
remix_and_resample effect optimization

### ISSUE THREAD

**Issue by wyz-honey** [livekit/agents#294]
```

                if isinstance(data, rtc.AudioFrame):
                    # TODO(theomonnom): The remix_and_resample method is low quality
                    # and should be replaced with a continuous resampling
                    frame = data.remix_and_resample(
                        self._sample_rate, self._num_channels
                    )

```

When will this TODO be optimized? After testing, the accuracy of this solution is much different from that of using pyaudio to read data from the microphone.

**Comment by davidzhao** [comment]
yeah we are working on it

**Comment by wyz-honey** [comment]
Great job, looking forward to it, thank you

**Comment by ChrisFeldmeier** [comment]
+1

**Comment by theomonnom** [comment]
Hey, sorry for the late reply. 
We released a high-quality resampler in rtc-v0.17.1 (using SoX)

https://github.com/livekit/python-sdks/releases/tag/rtc-v0.17.1
